src/main/java/net/sacredlabyrinth/phaed/simpleclans/managers/RequestManager.java

```diff
@@ -226,8 +225,10 @@ public void processInvite(Request req, VoteResult vote) {
         if (vote.equals(VoteResult.ACCEPT)) {
             ClanPlayer cp = plugin.getClanManager().getCreateClanPlayer(invited.getUniqueId());
             int maxMembers = plugin.getSettingsManager().getInt(CLAN_MAX_MEMBERS);
+            int unverifiedMaxMembers = plugin.getSettingsManager().getInt(CLAN_UNVERIFIED_MAX_MEMBERS);
 
-            if (maxMembers > 0 && maxMembers > clan.getSize()) {
+            if ((clan.isVerified() && maxMembers > 0 && maxMembers > clan.getSize()) ||
```

Comment:
Hmmm the only way I can think of doing that is by deleting the extra checks but I don't know

Reply:
Your condition is really too complex, you have to simplify it.
This is what I can see:
```java
if (clan.getSize() > maxMembers || (!clan.isVerified() && clan.getSize() > unverifiedMaxMembers)) {
    ChatBlock.sendMessageKey(invited, "this.clan.has.reached.the.member.limit");
    return;
}

ChatBlock.sendMessageKey(invited, "accepted.invitation", clan.getName());
clan.addBb(lang("joined.the.clan", invited.getName()));
plugin.getClanManager().serverAnnounce(lang("has.joined", invited.getName(), clan.getName()));
clan.addPlayerToClan(cp);
```

Reply:
> Your condition is really too complex, you have to simplify it. This is what I can see:
I think this would look cleaner if the first part was "clan.isVerified()" instead of the second part, what do you think? I mean something like this 

`if ((clan.isVerified() && maxMembers > 0 && maxMembers > clan.GetSize()) || (unverifiedMaxMembers > 0 && unverifiedMaxMembers > clan.getSize))`

I think it just looks better but I'll gladly accept your suggestion

Reply:
```java
int maxMembers = clan.isVerified() ? plugin.getSettingsManager().getInt(CLAN_MAX_MEMBERS) : plugin.getSettingsManager().getInt(CLAN_UNVERIFIED_MAX_MEMBERS);

//if remains the same
```


What do you think?

Reply:
Hmm, so instead of changing 'if' at all, change the original int. Sounds great, fixes the issue of complex expression.

Reply:
Exactly. (Edited the code above)